dbtmetabase/__init__.py

```diff
@@ -78,58 +139,103 @@ def main(args: list = None):
     )
     parser.add_argument(
         "--mb_https",
-        metavar="HTTPS",
-        type=bool,
-        default=True,
-        help="use HTTPS to connect to Metabase instead of HTTP",
+        action="store_true",
+        default=False,
+        help="Use HTTPS to connect to Metabase instead of HTTP",
+    )
+    parser.add_argument(
+        "--mb_verify",
+        metavar="CERT",
+        help="Path to certificate bundle used by Metabase client",
+    )
+    parser.add_argument(
+        "--database",
+        metavar="ALIAS",
+        required=True,
+        help="Target database name as set in Metabase (typically aliased)",
+    )
+    parser.add_argument(
+        "--dbt_database",
+        metavar="DB",
+        required=True,
+        help="Target database name as specified in dbt",
     )
     parser.add_argument(
-        "--database", metavar="DB", required=True, help="target database name"
+        "--schema",
+        metavar="SCHEMA",
+        help="Target schema name. Cannot be specified with --schema_excludes.",
     )
     parser.add_argument(
-        "--schema", metavar="SCHEMA", required=True, help="target schema name"
+        "--schema_excludes",
+        help="Target schemas to exclude. Cannot be specified with --schema. Will sync all schemas not excluded.",
     )
     parser.add_argument(
         "--sync",
-        metavar="ENABLE",
-        type=bool,
-        default=True,
-        help="synchronize Metabase database before export",
+        action="store_true",
+        default=False,
```

Comment:
Also here the default changed from `True` to `False`, or is it me misunderstanding `action="store_true"`? 😅

Reply:
Thanks for the catch. Regressed this change for the same reason as above. 
Want consistency in expected function. resolving comment